src/i18n/resources/en/login.json

```diff
@@ -34,7 +34,11 @@
   "password": "Password",
   "passwordChanged": "Password has been changed",
   "passwordError": "Password must contain six characters or more and have at least one lower case and one upper case alphabetical character and one number",
+  "passwordLowercase": "Password must contain at least one lowercase letter",
```

Comment:
@sorja im also wondering, isn't it better for the users to see all requirements in one message (like before )? Otherwise while they tap , they might see always different messages ...

Reply:
Sure thing